Admin/View/AutomationViewBuilder.php

```diff
@@ -44,6 +47,18 @@ public function setEntityClass(string $entityClass): AutomationViewBuilderInterf
         $this->addRequestParameter('entityClass', $entityClass);
         $this->addMetadataRequestParameters(['entityClass' => $entityClass]);
 
+        $this
+            ->addTabBadges([
+                static::BADGE_KEY => (new Badge('sulu_automation.get_task_amount', '/amount', 'value != 0'))
```

Comment:
`amount` usually means that something cannot be counted easily - would use `count`